kuma/javascript/src/a11y/a11y-nav.jsx

```diff
@@ -1,23 +1,64 @@
 // @flow
 import * as React from 'react';
+import { useContext } from 'react';
+
+import GAProvider from '../ga-provider.jsx';
 
 import { gettext } from '../l10n.js';
 
 export default function A11yNav() {
+    const ga = useContext(GAProvider.context);
+
+    /**
+     * Send a signal to GA when there is an interaction on one
+     * of the access menu links.
+     * @param {String} action - The action of the GA event
+     * @param {String} label  - The label for the GA event
+     */
+    const sendAccessMenuItemClick = (action, label) => {
+        ga('send', {
+            hitType: 'event',
+            eventCategory: 'Access Links',
+            eventAction: action,
+            eventLabel: label
+        });
+    };
+
     return (
         <ul id="nav-access">
             <li>
-                <a id="skip-main" href="#content">
+                <a
+                    id="skip-main"
+                    href="#content"
+                    onClick={() => {
+                        sendAccessMenuItemClick(
+                            'Skip to main content',
+                            '#content'
+                        );
+                    }}
+                >
                     {gettext('Skip to main content')}
```

Comment:
In [wiki.js](https://github.com/mdn/kuma/blob/a3d6cb8e7314b70e6c07a54a7c8d86d6c7682448/kuma/static/js/wiki.js#L95) we're only tracking `contextmenu` clicks. You have it on regular clicks. 
However, I put some `console.log` in on that event handler and it doesn't fire.

Reply:
Interesting, I definitely saw the GA events in the console. I was using the enter key, but should not make a difference right?

Reply:
> In [wiki.js](https://github.com/mdn/kuma/blob/a3d6cb8e7314b70e6c07a54a7c8d86d6c7682448/kuma/static/js/wiki.js#L95) we're only tracking `contextmenu` clicks.

Hmmm, does that not track both? i.e. click and contextmenu(i.e. right click)

Reply:
Looks to be the case: https://api.jquery.com/on/

Reply:
Made the changes here and tested again using the enter key as well as clicking the links, and the GA events are definitely being fired:

<img width="1674" alt="Screenshot 2019-12-18 at 12 23 05" src="https://user-images.githubusercontent.com/10350960/71078897-edfdda80-2192-11ea-9ab8-2da97695dfc2.png">